puppet/modules/slave/manifests/init.pp

```diff
@@ -251,6 +251,12 @@
     package { 'google-chrome-stable':
       ensure => latest,
     }
+
+    include ::chromedriver
+
+    class { 'chromedriver':
```

Comment:
Also, you can't both include it and define it as a class. That's redundant.

Reply:
Sorry, leftover from testing